### PR TITLE
Fix markdown link and replacing the URL for create issues.

### PR DESCRIPTION
**Pull request by rn4n**
Hello ✋ 

This PR fix the link for RFC 2119 and change the URL for creating new issues.

The link was changed because that repository does not have attention enough. This problem related about the RFC link was reported 4 years ago ([reference](https://github.com/wendtecnologia/semver/issues/9)) and the PR submitted last month ([reference](https://github.com/wendtecnologia/semver/pull/12)), neither of them was answered. So I think it's better to create news issues related about portuguese translation here.

### CODE REVIEW

**Comment by walkeralencar** [comment]
Fix markdown, OK
Replace URL, removing references from the group who translated to pt-br: Not OK.

**Comment by rn4n** [comment]
**EDIT**: _message sent before @walkeralencar deleted his message saying that this PR is invalid for taking credit from those who had participated in the translation of the document:_

> Wut? No reference about who made the translation was removed:
> 
> https://github.com/semver/semver.org/blob/9fba430ae67d3a88d37e20f4be8edb7ab2809942/lang/pt-BR/index.md#sobre
> 
> Only changed the link from where new issues related to portuguese documentation should be created. And the reason is simple: issues and pull requests in the repository linked previously do not have the proper attention. As an example, the issue that this pull request fix was reported 4 years ago.

**Comment by walkeralencar** [comment]
@reebr, in a lot of others repositories and managements, a fix of 1 character, was by-passed on my e-mail, sorry for that.

Your PR was properly merged, thanks.

**Comment by rn4n** [comment]
It's interesting how all other translations use https://github.com/mojombo/semver/issues as URL for new issues. Only the Portuguese translation uses the repository of a private company to do so.

**Comment by walkeralencar** [comment]
> It's interesting how all other translations use https://github.com/mojombo/semver/issues as URL for new issues. Only the Portuguese translation uses the repository of a private company to do so.

It's because, a lot of people joined as a community to translate this sṕecification 7 years ago, and to have discussions in Portuguese to be consolidated at main repo where everyone speaks English.

It's community work, started in a DOJO day about versioning in Continuous Integration, what is the problem if a private company started, translated and shared the translation? 

See about how the PR becomes to the main repo: https://github.com/semver/semver/issues/184

As always, we are open to suggestions.

Best Regards,

File: lang/pt-BR/index.md [hunk 1]
```diff
@@ -286,7 +286,7 @@ participação de:
 Toda colaboração na tradução pode ser acompanhada no link:
 http://pad.okfn.org/p/Fh9hjBPVu9
 
-Caso queira deixar sua opinião, por favor [abra uma issue no GitHub](https://github.com/semver/semver.org/issues).
+Caso queira deixar sua opinião, por favor [abra uma issue no GitHub](https://github.com/mojombo/semver/issues).
```
Comment (walkeralencar) [inline lang/pt-BR/index.md:289]: mojombo/semver will be redirected to semver/semver, adjust to remove only .org.

**Review by walkeralencar — CHANGES_REQUESTED** [review]
mojombo/semver will be redirected to semver/semver, adjust to remove only _.org_

**Review by walkeralencar — COMMENTED** [review]
Unfortunately, not every Brazilian speak English, so, the repo wendtecnologia/semver is for opinion and suggests in portuguese, not for ISSUES.
But, if this is a problem for you, we talk in Portuguese in the main repo. PS.: I don't recommend that.

**Comment by walkeralencar** [comment]
> **EDIT**: _message sent before @walkeralencar deleted his message saying that this PR is invalid for taking credit from those who had participated in the translation of the document:_
> 
> > Wut? No reference about who made the translation was removed:
> > https://github.com/semver/semver.org/blob/9fba430ae67d3a88d37e20f4be8edb7ab2809942/lang/pt-BR/index.md#sobre
> > Only changed the link from where new issues related to portuguese documentation should be created. And the reason is simple: issues and pull requests in the repository linked previously do not have the proper attention. As an example, the issue that this pull request fix was reported 4 years ago.

Yes, I've removed the message, because I recognized my mistake.

File: lang/pt-BR/index.md [hunk 2]
```diff
@@ -287,7 +286,7 @@ participação de:
 Toda colaboração na tradução pode ser acompanhada no link:
 http://pad.okfn.org/p/Fh9hjBPVu9
 
-Caso queira deixar sua opinião, por favor [abra uma issue no GitHub](https://github.com/wendtecnologia/semver/issues).
+Caso queira deixar sua opinião, por favor [abra uma issue no GitHub](https://github.com/mojombo/semver/issues).
```
Comment (alexandrtovmach) [inline lang/pt-BR/index.md:289]: Should it be replaced with `https://github.com/semver/semver.org/issues`?

Comment (rn4n) [reply lang/pt-BR/index.md:289]: @alexandrtovmach I don't know. Do you think that I should update this link?

Comment (alexandrtovmach) [reply lang/pt-BR/index.md:289]: Based on original content we should use `semver/semver.org`:
https://github.com/semver/semver.org/commit/b5e2e75374611657c0c62e0bcc8d03172d49e3d5#diff-c2d2fbe9e5c48c67f69cc53d17e084daR357

If you'd like to update your PR with that - it'll be great 👍

Comment (rn4n) [reply lang/pt-BR/index.md:289]:
```suggestion
Caso queira deixar sua opinião, por favor [abra uma issue no GitHub](https://github.com/semver/semver.org/issues).
```

Comment (rn4n) [reply lang/pt-BR/index.md:289]: @alexandrtovmach I'll create another PR because I've deleted the repository.

**Comment by rn4n** [comment]
Closing it in favor of #273.

FYI @alexandrtovmach.

**Pull request closed without merging**